cli/cmd/send_event_approvalFinished.go

```diff
@@ -36,11 +36,11 @@ var approvalFinishedCmd = &cobra.Command{
 	Use: "approval.finished",
 	Short: "Sends an approval.finished event to Keptn in order to confirm an open approval " +
 		"with the specified ID in the provided project and stage",
-	Long: `Sends an approval.finished event to Keptn in order to confirm an open approval
-with the specified ID in the provided project and stage. 
+	Long: `Sends an approval.finished event to Keptn in order to confirm an open approval with the specified ID in the provided project and stage. 
 
-This command takes the project (*--project*), stage (*--stage*). Besides, it is necessary to specify the ID (*--id*) of the corresponding approval.triggered event.
-The open approval.triggered events and their ID can be retrieved using the "keptn get event approval.triggered --project=<project> --stage=<stage>"" command."
+* This command takes the project (*--project*), stage (*--stage*). 
+* It is necessary to specify the ID (*--id*) of the corresponding approval.triggered event.
```

Comment:
Providing the id is optional. If the id is not provided, the user enters the interactive variant of this command.